client/src/Components/Common/Header/Header.js

```diff
@@ -14,6 +15,17 @@ const Header = (props) => {
         <Typography noWrap className={classes.title}>
           {pageName}
         </Typography>
+        { pageName==='Search'?
+         <IconButton
+         edge="start"
+         className={classes.filterButton}
+         color="#DADADA"
```

Comment:
Might be able to drop this attribute, looks like color is defined in the Style.js